2996_reflection/reflection.md

```diff
@@ -3747,7 +3829,7 @@ is interpreted as follows:
 
   * [#.#]{.pnum} If the `$id-expression$` denotes an overload set `$S$` and overload resolution for the expression `&$S$` determines a unique function `$F$` ([over.over]{.sref}), the `$reflect-expression$` represents `$F$`.
 
-  * [#.#]{.pnum} Otherwise, if the `$id-expression$` denotes a variable, structured binding, enumerator, or non-static data member or member function, the `$reflect-expression$` represents that entity.
+  * [#.#]{.pnum} Otherwise, if the `$id-expression$` denotes a variable or structured binding not captured by an enclosing `$lambda-expression$`, an enumerator, or a non-static data member or member function, the `$reflect-expression$` represents that entity.
```

Comment:
Would prefer splitting out the capturing part.

Reply:
Right, thanks for the reminder. Done.